app/src/main/java/com/coolapps/yo/tuto4u/HomeActivity.java

```diff
@@ -1,12 +1,31 @@
 package com.coolapps.yo.tuto4u;
 
 import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class HomeActivity extends BaseActivity {
 
+    RecyclerView mockTutorRecycler;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_home);
+        List<TutorData> info = new ArrayList<>();
+        for (int i = 0; i < 100; i++) {
+            TutorData t_data = new TutorData();
+            t_data.setText1("Tutor " + i);
+            t_data.setText2("Profession " + i);
+            t_data.setImage(R.mipmap.ic_launcher_round);
+            info.add(t_data);
+        }
```

Comment:
Could you please move this mocked logic to a new class, maybe call it a `TutorDataPresenter`. And there add a method named `getMockTutorsData()` and it will return `List< TutorData >`. So that when the actual data comes we can simply delete this mock method. This way we will not have to make any changes in `HomeActivity` later on. Thank you.